WORKLIST.txt

```diff
@@ -48,6 +49,10 @@ PLAYER:
 - equipped items are used to make attacks and to defend against attacks (raise AC, raise to-hit and damage stats, &c)
 - equippable items restricted to certain subset of all items ITEM_FLAG_EQUIPMENT, instead of hardcoded in CPlayer::Wield.
 - implement paginated inventory
+- how to remove cursed?
+- how to shoot bows
+- Different character classes
+- Magic spells etc?
```

Comment:
yes of course, the game I'm trying to copy has
mage
priest
druid
bard
and then multi-class (ranger is warrior/mage, &c)
spells come in 4 books per spec